formiodata/components.py

```diff
@@ -55,6 +64,10 @@ def value(self, value):
     def hidden(self):
         return self.raw.get('hidden')
 
+    def render(self):
+        self.html = f"<p>{self.form.get('value')}</p>"
```

Comment:
Isn't the `render` function too opinionated in the base `Component` class?
Or is it meant to be overriden by some child Components?
Please explain your idea, seems like a nice addition anyway!

Reply:
yes the target is  to override render by some child Components

Reply:
What about the already present `html` property in some core Formio components?
Like: https://github.com/novacode-nl/python-formio-data/blob/master/tests/data/test_example_builder.json#L9

I expect the default `render` function overrides this one now?
Shouldn't you check whether there's already a `html` property present and decide whether to override?

Reply:
[updated](https://github.com/novacode-nl/python-formio-data/pull/3/commits/865b8157570961c3755989d7d434825a41c7678c)